trantor/net/inner/Connector.cc

```diff
@@ -27,6 +27,18 @@ Connector::Connector(EventLoop *loop, InetAddress &&addr, bool retry)
 {
 }
 
+Connector::~Connector()
+{
+    if (socketHanded_ == false && fd_ != -1)
```

Comment:
Should this variable be named socketHandled_?

Reply:
I named it `Handed` because the lifetime is handed to IO Channel. We should not attempt to manage it ourselves.